pkg/configuration/configuration.go

```diff
@@ -48,6 +48,22 @@ type AcceptedKernelTaintsInfo struct {
 	Module string `yaml:"module" json:"module"`
 }
 
+// SkipScalingTestDeploymentNamesInfo contains a list of names of deployments that should be skipped by the scaling tests to prevent issues
+type SkipScalingTestDeploymentNamesInfo struct {
```

Comment:
Minor suggestion: Since the "Name" word itself is one field of the struct, I think it can be removed from the type name.
```
type SkipScalingTestDeployments struct {
  Name ...
  Namespace ...
}
```